src/rcpp_octave.cpp

```diff
@@ -506,18 +554,23 @@ octave_value octave_feval(const string& fname, const octave_value_list& args, in
 #endif
 	){
 		REprintf(R_PACKAGE_NAME" - Caught Octave exception: interrupt\n");
-		recover_from_exception();
+		recover_from_exception_rcppoct();
 		REprintf("\n");
 		//error_state = -2;
 	}
 	catch (std::bad_alloc)
 	{
 		REprintf(R_PACKAGE_NAME" - Caught Octave exception: bad_alloc\n");
-		recover_from_exception();
+		recover_from_exception_rcppoct();
 		REprintf("\n");
 		//error_state = -3;
+	} catch(const octave::execution_exception& e) {
+			std::ostringstream err;
+			err << R_PACKAGE_NAME" - Octave error: execution_exception";
```

Comment:
The exception is being caught here. Should the wording be improved?
See comment in related test in f3156b3b8210160f1298d57b01b97f5402543949.